test/quantization/core/test_quantized_op.py

```diff
@@ -5777,6 +5777,27 @@ def test_conv_reorder_issue_onednn(self):
             qx = torch.quantize_per_tensor(x, scale=1.0, zero_point=0, dtype=torch.quint8)
             # The following should pass when input shape is changed
             torch.ops.quantized.conv2d(qx, w_packed, output_scale=1.0, output_zero_point=0)
+        # conv_transposed part
```

Comment:
maybe split this into a separate test?

Reply:
Thanks for the comment. Put it into a separate test.